### ISSUE TITLE
Support for M and L AttributeValue

### ISSUE THREAD

**Issue by chrisronline** [Medium/dynamite#79]
I'm fairly new to DynamoDB and looking to migrate from our native aws library usage to this library.

I'm seeing an issue where our data types with attribute values `M` or `L` are throwing an unsupported error and it doesn't seem like [they are supported](https://github.com/Medium/dynamite/blob/master/lib/typeUtil.js#L144) but they exist in the [official docs](http://docs.aws.amazon.com/amazondynamodb/latest/APIReference/API_AttributeValue.html#DDB-Type-AttributeValue-M) as valid types.

Am I missing something?

Thanks

P.S. It seems it would be fairly simple to support, something like:
```
case 'M':
  var mapped = {};
  for (var k in obj.M) {
    mapped[k] = objectToValue(obj.M[k]);
  }
  return mapped;
case 'L':
  return obj.L.map(objectToValue);
```

**Comment by adrianlee44** [comment]
@chrisronline, would you like to create a PR for your changes?

**Comment by chrisronline** [comment]
Done ^^

**Comment by adrianlee44** [comment]
Thanks for your PR. Before I can merge your changes, can you sign our CLA (https://github.com/Medium/opensource/blob/master/sign-cla.md).

**Comment by chrisronline** [comment]
The md5 doesn't match for me - am I doing something wrong?

```
chrisronline$ git pull
Already up-to-date.
chrisronline$ git status
On branch master
Your branch is up-to-date with 'origin/master'.
chrisronline$ md5 cla-2.0.md 
MD5 (cla-2.0.md) = 143da42902fd29612270cc1e4c7b5296
```

From the CLA:
```
You can confirm the MD5 checksum of the CLA by running the md5 program over cla-2.0.md:

md5 cla-2.0.md
MD5 (cla-2.0.md) = 0962bd43bb8849553d91650a78cb2924
If the output is different from above, do not sign the CLA and let us know.
```

**Comment by adrianlee44** [comment]
@chrisronline, thanks for spotting the error. We have fixed the hash.

**Comment by chrisronline** [comment]
@adrianlee44 https://github.com/Medium/opensource/pull/96

**Comment by adrianlee44** [comment]
This is fixed with #79
